addons/html_editor/static/src/main/chatgpt/chatgpt_prompt_dialog.js

```diff
@@ -63,6 +64,11 @@ export class ChatGPTPromptDialog extends ChatGPTDialog {
         );
     }
 
+    /** @returns {string} */
+    get authorName() {
+        return this.message.author === "user" ? _t("You") : _t("OdooBot");
```

Comment:
Unrelated to your changes, but the hardcoded "OdooBot" is a bit weird to me. OdooBot is a user in the system that can change its name if wanted. So shouldn't they use the name of the user with ID 1 instead?